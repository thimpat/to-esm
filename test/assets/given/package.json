{
  "name": "something",
  "version": "1.0.2",
  "random": 123456,
  "main": "./example/cjs/demo.cjs",
  "module": "./generated/esm/demo.mjs",
  "type": "commonjs"
}