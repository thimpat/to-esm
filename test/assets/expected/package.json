{
  "name": "something",
  "version": "1.0.2",
  "random": 123456,
  "main": "./example/cjs/demo.cjs",
  "module": "./actual/demo-test-14.mjs",
  "type": "module",
  "exports": {
    "require": "./given/demo-test-14.cjs",
    "import": "./actual/demo-test-14.mjs"
  }
}